xgi/generators/uniform.py

```diff
@@ -286,10 +341,26 @@ def uniform_erdos_renyi_hypergraph(n, m, p, p_type="degree", seed=None):
     Hypergraph
         The Erdos Renyi hypergraph
 
-
     See Also
     --------
     ~xgi.generators.random.random_hypergraph
+
+    Notes
+    -----
+    Because XGI only stores edges as sets, if self-loops are allowed,
+    for example, the edge (0, 0, 0) will be mapped to {0}. However,
+    because this is explicitly a *uniform* method, we discard these edges
+    so that this is the case. For sparse networks, this is a rare
```

Comment:
```suggestion
    because this is explicitly a *uniform* method, we discard these edges. 
    For sparse networks, this is a rare
```
if "so that this is the case" meant "so that the output is a uniform hypergraph" then I think we can remove that bit.